src/pages/Assets/AssetDetails/AssetActions.tsx

```diff
@@ -4,21 +4,19 @@ import { AssetMarketData } from '@shapeshiftoss/market-service'
 import { useModal } from 'context/ModalProvider/ModalProvider'
 
 export const AssetActions = ({ asset }: { asset: AssetMarketData }) => {
-  const modal = useModal()
+  const { send, receive } = useModal()
+  const { open: openSend } = send
+  const { open: openReceive } = receive
   return (
     <ButtonGroup
       ml={{ base: 0, lg: 'auto' }}
       mt={{ base: 6, lg: 0 }}
       width={{ base: 'full', lg: 'auto' }}
     >
-      <Button onClick={() => modal.open('send')} isFullWidth leftIcon={<ArrowUpIcon />}>
+      <Button onClick={() => openSend()} isFullWidth leftIcon={<ArrowUpIcon />}>
```

Comment:
Shouldn't this be passing asset as a prop?
```suggestion
      <Button onClick={() => openSend({ asset )} isFullWidth leftIcon={<ArrowUpIcon />}>
```

Reply:
as discussed in this branch send modal is unimplemented